src/legacy/core_plugins/kibana/public/visualize/listing/visualize_listing.js

```diff
@@ -53,6 +53,10 @@ export function VisualizeListingController($injector, createNewVis) {
     this.showNewVisModal = true;
   };
 
+  this.edit = ({ id }) => {
+    kbnUrl.change(`${VisualizeConstants.EDIT_PATH}/{{id}}`, { id });
```

Comment:
Maybe move this to `visualize_constants.ts` to align with `dashboard_constants.ts`?

Reply:
done